diff --git a/lib/src/jpeg-8b/jconfig.h b/lib/src/jpeg-8b/jconfig.h
index 3b1db7d..966b1d5 100644
--- a/lib/src/jpeg-8b/jconfig.h
+++ b/lib/src/jpeg-8b/jconfig.h
@@ -46,7 +46,7 @@ typedef unsigned char boolean;
 
 /* #undef TWO_FILE_COMMANDLINE */
 /* #undef NEED_SIGNAL_CATCHER */
-#define DONT_USE_B_MODE 1
+/* #undef DONT_USE_B_MODE */
 
 /* Define this if you want percent-done progress reports from cjpeg/djpeg. */
 /* #undef PROGRESS_REPORT */
